Formula/q/qt.rb

```diff
@@ -287,6 +302,18 @@ def install
       libexec.install app
       bin.write_exec_script libexec/app.basename/"Contents/MacOS"/app.stem
     end
+
+    # Modify unofficial pkg-config files to fix up paths and use frameworks.
+    # Also move them to `libexec` as they are not guaranteed to work for users,
+    # i.e. there is no upstream or Homebrew support.
+    lib.glob("pkgconfig/*.pc") do |pc|
+      inreplace pc do |s|
+        s.gsub! " -L${libdir}", " -F${libdir}", false
+        s.gsub! " -lQt6", " -framework Qt", false
+        s.gsub! " -Ilib/", " -I${libdir}/", false
+      end
+      (libexec/"lib/pkgconfig").install pc
```

Comment:
I am trying to use these pkgconfig files in the CI of Octave. But I'm struggling to understand where they are installed.
They don't seem to be present in either `${HOMEBREW_PREFIX}/opt/qt@6/lib/pkgconfig` or `${HOMEBREW_PREFIX}/lib/pkgconfig`.

What does the `libexec` part entail here?
Looking at https://docs.brew.sh/Formula-Cookbook, does it mean that they are installed in `${HOMEBREW_PREFIX}/opt/qt@6/libexec/lib/pkgconfig`? If they are, that looks like an odd choice.

Reply:
They're installed in an odd location because our use of the `*.pc` files is an unsupported hack.

If you'd like to use them, I suggest filing an issue with the Qt project so that they can provide proper `*.pc` files instead of the hacked up ones we've included here. (If you're in a pinch, using `find` may also help, but [you're on your own here](https://github.com/Homebrew/homebrew-core/blob/6b9f22abafb887533d9dc5dd06f604b52ba7c24e/Formula/q/qt.rb#L307-L308).)

Reply:
Thank you for clarifying.

I don't have access to a Mac computer. So, I'll probably not be able to create a useful report for upstream Qt.